app/views/ClientIdView.scala.html

```diff
@@ -66,5 +71,5 @@ <h1 class="govuk-heading-l">Client ID</h1>
   <p class="govuk-body">Generate a client secret and use it with your client ID to test your application in the sandbox.</p>
 
   <a id="continue" class="govuk-button" href="@controllers.routes.Credentials.clientSecrets(app.id)">Continue</a>
-  <script src='@Assets.versioned("javascripts/copy-button.js")' type="text/javascript"></script> 
+  <!-- <script src='@Assets.versioned("javascripts/copy-button.js")' type="text/javascript"></script>  -->
```

Comment:
Isn't this line replaced further up and this commented out code no longer needed?

Reply:
Yes I'll fix this